python/lsst/cp/pipe/cpDark.py

```diff
@@ -114,19 +124,38 @@ def run(self, inputExp):
         # exposure, if possible.  Otherwise, use the cameraGeom value.
         gains = self._get_gains(inputExp)
 
+        # Is this gainContext load bearing?
```

Comment:
I don't understand this comment.

Reply:
Cosmic ray rejection was highly dependent on this `gainContext` with the old ISR task, and without this scaling, we'd end up with too many CRs.  I've filed a ticket to see if the improvements in the new task make this `gainContext` unnecessary or not.  DM-48754 (with comment updated to point there).